### PR TITLE
Mobile portifolio

### PR DESCRIPTION
**Pull request by EyasuTeshome**
I have added two sections in index.html file. the sections are "work" and "about-me". I have added style to the two sections added on index.html. Edited readme.md, added images that are required to finish the task.

### CODE REVIEW

File: img/index.html [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+            </div>
+            <div class="button">
+                <p>
+                    See Project
+                </p>
```
Comment (tufoinnkuo10) [inline img/index.html:74]: - [ ] please kindly use a `button tag` here . make the hover and style to be like the figma version

File: img/index.html [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+              <ul>
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+              </ul>
+          </div>
+          <div class="button">
+              <p>
+                  See Project
+              </p>
```
Comment (tufoinnkuo10) [inline img/index.html:105]: - [ ]button tag should be used here as well

File: img/index.html [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
```
Comment (tufoinnkuo10) [inline img/index.html:135]: - [ ]button

File: img/index.html [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+    </section>
+    <section class="about-myself">
+      <div class="about-header">
+        <h2>About <br> Myself</h2>
+      </div>
+      <div class="about-content">
+        <p class="about-content-p">
+          Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <p class="connect">LET'S CONNECT</p>
```
Comment (tufoinnkuo10) [inline img/index.html:177]: - [ ] please use an `h tag` here as it is a subheading

File: index.html [hunk 5]
```diff
@@ -29,25 +32,192 @@ <h1>
...
+        <p class="about-content-p">
+          Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <p class="connect">LET'S CONNECT</p>
+        <a href="twitter.com"><img src="img/twitter.png" alt=""></a>
+        <a href="linkedin.com"><img src="img/Linkedin_icon.png" alt=""></a>
+        <a href="medium.com"><img src="img/medium.png" alt=""></a>
+        <a href="github.com"><img src="img/github.png" alt=""></a>
+        <a href="angellist.com"><img src="img/angellist.png" alt=""></a>
+        <button type="button" class="my-button">Get My Resume</button>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] kind use an `ul tag` here and for each icon, use a `li`

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @EyasuTeshome 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅cards are well structured

Required Changes ♻️
Check the comments under the review

tufoin
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: img/index.html (hunk 2 above)
Comment (sja-thedude) [reply img/index.html:105]: Implemented 👍

File: img/index.html (hunk 3 above)
Comment (sja-thedude) [reply img/index.html:135]: Implemented 👍

File: index.html (hunk 5 above)
Comment (sja-thedude) [reply index.html]: Implemented 👍

File: img/index.html (hunk 4 above)
Comment (sja-thedude) [reply img/index.html:177]: Implemented 👍

File: index.html [hunk 6]
```diff
@@ -69,9 +81,7 @@ <h2>Tonic</h2>
                 </ul>
             </div>
             <div class="button">
-                <p>
-                    See Project
-                </p>
+                <button type="button" class="work-button">See Project</button>
```
Comment (sja-thedude) [inline index.html:84]: - [ ] As suggested by the previous reviewer, kindly make the hover effect on all the buttons and style be like the Figma version. Kindly make sure that when you hover on to any button in the project the color changes as the design.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @EyasuTeshome, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -5,19 +5,22 @@
...
     <header>
         <a href="#home">Eyasu</a>
+
+
         <nav>
-            <img class="nav-pic" src="./img/Union.png" alt="">
-            <a href="#portifolio">Home</a>
-            <a href="#about">About</a>
-            <a href="#contact">Contact</a>
+            <img class="nav-pic" src="img/union.png" alt="">
```
Comment (SajjadAhmad14) [inline index.html:21]: [ OPTIONAL BUT RECOMMENDED ] Hamburger does not show on mobile view. Kindly make sure that there is a hamburger menu for navigation on the right side of the header.

hamburger

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Josh, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**